test/lib/token.test.js

```diff
@@ -24,24 +27,45 @@ describe('Token Model', () => {
         datastore = {
             update: sinon.stub()
         };
+        ironMock = {
+            seal: sinon.stub(),
+            unseal: sinon.stub(),
+            defaults: 'defaults'
+        };
     });
 
     beforeEach(() => {
-        datastore.update.resolves({});
-
         createConfig = {
             datastore,
             userId: 12345,
-            uuid: '1a2b3c',
+            value: sealed,
             id: 6789,
             name: 'Mobile client auth token',
             description: 'For the mobile app',
             lastUsed: '2017-05-10T01:49:59.327Z',
             password
         };
+
+        datastore.update.resolves({});
+        ironMock.seal.yieldsAsync(null, sealed);
+        ironMock.unseal.yieldsAsync(null, unsealed);
+
+        mockery.registerMock('iron', ironMock);
```

Comment:
You want to be mocking the `sealField` dependency instead of the `iron` dependency that is used in `sealField`.